packages/ibm-products/src/components/PageHeader/PageHeader.js

```diff
@@ -81,7 +81,6 @@ export let PageHeader = React.forwardRef(
       tags,
       title,
       withoutBackground,
-      breadcrumbOverflowTooltipAlign = defaults.breadcrumbOverflowTooltipAlign,
```

Comment:
Don’t think we should be removing this here? This and the `overflowTooltipAlign` below were both recently added from @makafsal’s PR #4907.